kumascript/index.js

```diff
@@ -0,0 +1,31 @@
+const renderMacros = require("./src/render.js");
+const Templates = require("./src/templates.js");
+const AllPagesInfo = require("./src/info.js");
+
+class Renderer {
+  constructor() {
+    this.allPagesInfo = null;
+    this.templates = new Templates();
+  }
+
+  use(pageInfoByUri) {
```

Comment:
Does there need to be an `update` method as well?

https://github.com/mdn/stumptown-renderer/issues/472 is about the fact that `this.allTitles` can and will mutate. It's closely linked to https://github.com/mdn/stumptown-renderer/issues/466 which is about consecutive builds when you restart it on the terminal. 

Yeah, we're going to need to mutate `this.allPagesInfo` too. We'll just have to make a mental note to remember to implement that.

Reply:
The `use()` method can be called again whenever `this.allTitles` changes, and the render will use the updated values, so I think we're good here!